integration-test/pom.xml

```diff
@@ -64,6 +90,33 @@
 
     <dependency>
       <groupId>${project.groupId}</groupId>
+      <artifactId>livy-integration-test-minicluster-dependencies_2.10</artifactId>
```

Comment:
minicluster-dependencies projects are special. They don't produce a jar. Also, <exclusions> tags below removes all their dependencies. They add nothing to the classpath.
This's also a build dependency to make sure minicluster-dependencies submodules are processed before integration-test.